src/Stratis.Bitcoin/Configuration/NodeSettings.cs

```diff
@@ -296,6 +296,8 @@ public NodeSettings()
                 nodeSettings.ConnectionManager.ExternalEndpoint = new IPEndPoint(IPAddress.Loopback, nodeSettings.Network.DefaultPort);
             }
 
+            nodeSettings.ConnectionManager.BanTime = config.GetOrDefault<int>("bantime", ConnectionManagerSettings.DefaultMisbehavingBantime);
```

Comment:
if this is a new option we support, it should go to PrintHelp as well

Reply:
yes done